app/templates/layouts/_base.html

```diff
@@ -33,7 +33,18 @@
     <![endif]-->
     {% endblock styles -%}
     {% endblock head -%}
-    {{ analytics }}
+    <!-- Google Analytics -->
+    <script>
```

Comment:
I wonder if we should put this into a separate template and include it here to keep it separate?